<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinMeetingDialog</class>
 <widget class="QDialog" name="JoinMeetingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Join Meeting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #555879
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>700</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 32px
}</string>
   </property>
   <property name="text">
    <string>Join Meeting</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalWidget" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>400</width>
     <height>150</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="meetingIdField">
      <property name="styleSheet">
       <string notr="true">QComboBox {
	color: white;
	border: 2px solid #DED3C4;
	border-radius: 10px;
	padding: 10px 25px;
	background: transparent;
}

QComboBox:on {
  border-bottom-left-radius: 0px;
  border-bottom-right-radius: 0px;
}

QComboBox::drop-down {
    margin-right: 10px;
}

QComboBox QAbstractItemView {
	border: 2px solid #DED3C4;
	border-bottom-left-radius: 10px;
	border-bottom-right-radius: 10px;
	selection-background-color: #555879;
	selection-color: #DED3C4;
	background-color: #555879;
}

QComboBox QAbstractItemView::item {
	padding: 5px;
	color: #DED3C4;
}

QComboBox QAbstractItemView::item:hover {
	background-color: #6B6E8C;
	color: #FFFFFF;
}

QComboBox QAbstractItemView::item:selected {
	font-weight: bold;
}</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Meeting ID*</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="passwordField">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
  color: white;
  border: 2px solid #DED3C4;
  border-radius: 10px;
  padding: 10px 25px;
  background: transparent;
}

QLineEdit:focus {
  border: 2px solid #F4EBD3;
}</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhHiddenText|Qt::ImhLatinOnly|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Password*</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showPwdIcon">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {
        width: 0;
        height: 0;
    }
    QCheckBox {
        spacing: 0;
        margin: 0;
    }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="joinMeetBtn">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  color: #555879;
  padding: 0.2em 1.7em;
  font-size: 14px;
  border-radius: 0.5em;
  background: #F4EBD3;
  border: 1px solid #DED3C4;
}

QPushButton:hover {
  background: rgb(230, 221, 209);
}

QPushButton:pressed {
  color: #98A1BC;
  background: #DED3C4;
}
</string>
      </property>
      <property name="text">
       <string>Join Meeting</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
